README.md

```diff
@@ -34,6 +33,73 @@ Special notes:
  * Since we cannot distribute Java Richards, a patch is required to derive the
    tracing version (`patches/trace_java_richards.diff`)
 
+## Using the scripts here to analyse an existing benchmark
+
+### Mandatory requirements
+
+  * Python 2.7 (the code here is not Python 3.x ready)
+  * bzip2 / bunzip2
+  * Python modules: numpy, rpy2
+  * R
+
+### Optional requirements
+
+  * Python modules required for plotting: matplotlib, seaborn
+  * Required for generating LaTeX tables: a LaTeX distribution which provides
+    pdflatex, and the following packages: amsmath, amssymb, booktabs, calc,
+    geometry, mathtools, multicol, multirow, rotating, sparklines, xspace.
+    The TeX Live distribution should be fine.
+
+### Setting up R
+
+To run the scripts here, it is necessary to install some R packages. By default,
+R will install these packages in `$HOME/R`. If you do not want R to use your
+home directory, then set the environment variable `$R_LIBS_USER`, e.g. (in BASH):
+
+```bash
+$ git clone https://github.com/softdevteam/warmup_experiment.git
+$ cd warmup_experiment
+$ mkdir R
+$ export R_LIBS_USER=`pwd`/R
+$ echo "export R_LIBS_USER=`pwd`/R" >> ~/.bashrc
+```
+
+To install the necessary packages, open R on the command line, and run the
+following commands:
+
+```R
+> install.packages("devtools")
+> devtools::install_github("rkillick/changepoint")
```

Comment:
The workaround for this R bug is: `options(download.file.method = "wget")`. Let's mention it as there's about a 99% chance users will be using debian (which has the buggy R version).